src/i2c-dev.c

```diff
@@ -15,7 +15,7 @@
 
 #include "i2c-dev.h"
 
-extern inline __s32 i2c_smbus_access(int fd, char rw, int cmd, int size, union i2c_smbus_data *data) {
+extern  __s32 i2c_smbus_access(int fd, char rw, int cmd, int size, union i2c_smbus_data *data) {
```

Comment:
Are you sure there aren't any side effect?

Reply:
Well ,I can call them normally,and no warnings were reported at compile time

Reply:
Is the only issue a warning at compile time? The problem is that wiringX must work on various platforms and since where not testing on all of them i'd rather leave the warning in place then being uncertain about possible side effects.

Reply:
I have tested it on raspberrypi 3B+ , and it works , but for more platforms , you're right.

Reply:
Thank you for your patient guidance，again.